elixir/apps/domain/lib/domain/crypto.ex

```diff
@@ -3,6 +3,7 @@ defmodule Domain.Crypto do
 
   def psk do
     random_token(@wg_psk_length, encoder: :base64)
+    |> String.slice(0, @wg_psk_length)
```

Comment:
Just for my own curiosity was wondering why we shortened the random token?

Reply:
Wireguard requires PK to be of a certain length. We generate the required bits but after encoding them the length of the string will be more than desired (+~30%) so we shorten it back.

Reply:
Ah, ok, gotcha.  Yeah, I saw that after base64 encoding it was 44 bytes, but just wasn't sure why it needed to be shortened now.

Reply:
Ah here is the problem I noticed in https://github.com/firezone/firezone/pull/6761#discussion_r1805614902!